src/style/big-search-bar.css

```
@@ -0,0 +1,41 @@
+.big-search-bar {
+  display: flex;
+}
+
+.big-search-bar--margin {
+  margin-top: 20px;
+}
+
+.big-search-bar__input {
+  width: 100%;
+  height: 70px;
+  margin-bottom: 20px;
+  padding-left: 62px;
+  box-sizing: border-box;
+  border: solid 1px;
+  border-radius: 4px;
+  border-color: #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  cursor: pointer;
+
+  background-image: url(../images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px center;
+  background-size: 19px;
+
+  font-family: 'Avenir Book', sans-serif;
```

Move `font-family: 'Avenir Book', sans-serif;` into `html` selector
```suggestion
  font-family: inherit;
```